server/startup/cron.js

```diff
@@ -77,6 +79,11 @@ Meteor.startup(function() {
 			job: cleanupOEmbedCache,
 		});
 
+		new Agenda({
+			mongo: MongoInternals.defaultRemoteCollectionDriver().mongo.client.db(),
+			collection: 'rocketchat_agenda_jobs',
```

Comment:
I don't know what I did wrong, but the collection name is not being set. It always goes to the default `agendaJobs`.